lib/srv/sess.go

```diff
@@ -17,23 +17,27 @@ limitations under the License.
 package srv
 
 import (
+	"bytes"
 	"fmt"
 	"io"
-	"strings"
 	"sync"
 	"time"
 
 	"github.com/gravitational/teleport/lib/defaults"
 	"github.com/gravitational/teleport/lib/events"
-	"github.com/gravitational/teleport/lib/recorder"
 	rsession "github.com/gravitational/teleport/lib/session"
+	"github.com/gravitational/teleport/lib/sshutils"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/codahale/lunk"
 	"github.com/gravitational/trace"
 	"golang.org/x/crypto/ssh"
 )
 
+// lingerTTL is the default value for session's "LingerTTL" property
```

Comment:
comment is not very descriptive - kinda self refferential :) also, may be makes sense to move it to `defaults`